apps/labs/posts/nebari-typer-cli.md

```diff
@@ -0,0 +1,161 @@
+    />
+    <i>Output of new Nebari CLI help command</i>
+</p>
+
+Now, we will see the output of the `nebari init` help command to see all the arguments and options that are available to create and initialize a `nebari-config.yaml` file.
+
+```bash
+  nebari init --help
+```
+
+<p align="center">
+    <img
+     alt="Output of nebari init help command"
+     src="/posts/asmi-blogposts/nebari-init-help.png" />
+    <i>Output of nebari init help command</i>
+</p>
+
+Like we can see it has lots of options available to pass as a flag in the command while executing it. The `guided-init` flag in the init command is the most valuable flag. If we are using Nebari first time and we don't know exactly what other options and arguments are there for the `init` command then we can run `init` command using `--guided-init` flag. It's ask values from the user and it is more friendly survey or questionnaire for the user to fill out. Let's see the output of it.
+
+```bash
+  nebari init --guided-init
+```
+
+<p align="center">
+    <img
+     alt="Output of nebari init --guided-init command"
+     src="/posts/asmi-blogposts/nebari-gi.png" 
+    />
+    <img
+     alt="Output of nebari init --guided-init command"
+     src="/posts/asmi-blogposts/nebari-gi1.png" 
+     />
+    <img
+     alt="Output of nebari init --guided-init command"
+     src="/posts/asmi-blogposts/nebari-gi2.png" 
+     />
+    <i>Output of nebari init guided-init command</i>
+</p>
+
+As we can see it's more like a survey that asks users to fill out the correct details for generating the `nebari-config.yaml` file. We have used the [questionary library](https://github.com/tmbo/questionary) for having the options for the questions.
+
+After creating or generating the `nebari-config.yaml`, It's time to validate and render it respectively. First we will see the `help` command output for the validate to see all the options and requirements available for it.
+
+```bash
+  nebari validate --help && nebari render --help
+```
+
+<p align="center">
+    <img
+     alt="Output of nebari validate and render help"
+     src="/posts/asmi-blogposts/nebari-vrh.png" />
+    <i>Output of nebari validate and render help command</i>
+</p>
+
+As we can see we have to pass the `-c/--config` flag for validating the `nebari-config.yaml` file and we have to do the same for rendering the infrastructure using the render command.
+
+```bash
+  nebari validate -c qhub-config.yaml && nebari render -c qhub-config.yaml
+```
+
+<p align="center">
+    <img
+     alt="Output of nebari validate and neabri render"
+     src="/posts/asmi-blogposts/nebari-rv.png" />
+    <i>Output of nebari validate and render command</i>
+</p>
+
+Render command builds the infrasture for the deployment. It creates all terraform stages file that are required during deployment.
+
+**_NOTE:_** It's not neccessary to run validate and render command seperately. Because it runs under the hood of the deploy command.
```

Comment:
```suggestion
**_NOTE:_** It's not necessary to run the `validate` and `render` commands separately. Because it runs under the hood of the deploy command.
```
`+` the other suggestions